images/Makefile

```diff
@@ -5,19 +5,24 @@ IMAGE_REPO = ghcr.io/kamu-data
 KAMU_WEB_UI_VERSION = $(shell cat ../package.json | jq -r '.version')
 KAMU_WEB_UI_IMAGE = $(IMAGE_REPO)/kamu-web-ui
 
+KAMU_WEB_UI_MAJOR_VERSION = $(shell echo $(KAMU_WEB_UI_VERSION) | cut -d. -f 1)
+KAMU_WEB_UI_MINOR_VERSION = $(shell echo $(KAMU_WEB_UI_VERSION) | cut -d. -f 2)
+
 #########################################################################################
 
 .PHONY: kamu-web-ui
 kamu-web-ui:
 	docker build \
 		--build-arg KAMU_WEB_UI_VERSION=$(KAMU_WEB_UI_VERSION) \
 		-t $(KAMU_WEB_UI_IMAGE):$(KAMU_WEB_UI_VERSION) \
+		-t $(KAMU_WEB_UI_IMAGE):$(KAMU_WEB_UI_MAJOR_VERSION).$(KAMU_WEB_UI_MINOR_VERSION) \
 		kamu-web-ui/
 
 
 .PHONY: kamu-web-ui-push
 kamu-web-ui-push:
 	docker push $(KAMU_WEB_UI_IMAGE):$(KAMU_WEB_UI_VERSION)
+	docker push $(KAMU_WEB_UI_IMAGE):$(KAMU_WEB_UI_MAJOR_VERSION).$(KAMU_WEB_UI_MINOR_VERSION)
 	docker tag $(KAMU_WEB_UI_IMAGE):$(KAMU_WEB_UI_VERSION) $(KAMU_WEB_UI_IMAGE):latest
```

Comment:
@sergiimk Is there any particular reason this `latest` tag is added here and not during the build stage?

Reply:
No reason. Ideally I wanted to avoid producing a bunch of tags locally and only tag during push, but unfortunately docker does not allow something like `docker push <image> -t <name1> -t <name2> -t <name3>`.

Feel free to move to build phase for consistency.